canonicalwebteam/blog/logic.py

```diff
@@ -97,13 +97,15 @@ def transform_article(
             article["content"]["rendered"]
         )
 
-    if "_start_month" in article:
+    # if _start_month is set here we can assume that day and year
+    # will also have a value, same for the end data
```

Comment:
no need to comment a if statement.

And assuming is not a good pattern

Reply:
Do you prefer a test on all attributes here?

This is not documented in the [wordpress Rest docs](https://developer.wordpress.org/rest-api/reference/posts/) as far as I can tell. Seems to be done with a plugin.

Reply:
then what do you trust more?

Reply:
Alright, I think this speaks for a full check then? Updated it.